tools/wheels/repair_windows.sh

```diff
@@ -4,6 +4,17 @@ WHEEL="$1"
 DEST_DIR="$2"
 OPENBLAS_DIR=$(python -c"import scipy_openblas32 as sop; print(sop.get_lib_dir())")
 
+# Determine the strip command based on TARGET_ARCH
+# TARGET_ARCH should be set by the CI environment (e.g., ARM64, AMD64)
+TARGET_ARCH="${TARGET_ARCH:-}" # Default to empty string if not set
+STRIP_COMMAND="strip"
+if [ "$TARGET_ARCH" = "ARM64" ]; then
+  STRIP_COMMAND="llvm-strip"
+  echo "Using llvm-strip for ARM64 target."
```

Comment:
Are you able to check locally whether this stripping actually changes anything? IIRC the only reason we have it is because `mingw-w64` is leaving (or used to leave) some bad content at the end of each `.pyd` file. For LLVM-based compilers, this may well do nothing, since we're not building with debug info or anything like that.

Reply:
How about inserting debug output that just checks the file size right before and after stripping. `ls -l` on the .whl file?

Reply:
@rgommers yeah the size difference for x64 after strip is 14mb less, for arm it's 0.158kb bigger, almost identical. I guess we can count that as something we can just skip for arm right?